tools/kind.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+REPO_ROOT="${REPO_ROOT:-"$(realpath "$(dirname "${BASH_SOURCE[0]}")/..")"}"
+BUILD_ROOT="${REPO_ROOT}/build"
+BUILD_BIN="${BUILD_ROOT}/bin"
+
+
+NAME=kind
```

Comment:
curious if you considered `k3d` at all? at least a brief comparison of the two would be nice in the PR description or something.

Reply:
Yes I did a while back, I had something written down let me see If i can dig it up.